exercises/04-integration/README.md

```diff
@@ -382,6 +382,10 @@ In this exercise, we'll set up every endpoint to send saga audit messages to Ser
 
 If you are not familiar with the saga audit plugin, you will need to read the [documentation](https://docs.particular.net/nservicebus/sagas/saga-audit) for how to install it first.
```

Comment:
The URL should be changed to https://docs.particular.net/nservicebus/sagas/saga-audit?version=sagaaudit_2